src/one_dragon/envs/git_service.py

```
@@ -746,12 +768,8 @@ def _fetch_remote(
             log.error('拉取结束后无法恢复主仓库远程地址')
             return False
         if not success:
-            if (
-                not self._rebuilding_repository
-                and failure_is_missing_object
-                and all(failure_is_missing_object)
-            ):
-                log.warning('所有候选代码源均因对象缺失失败，开始自动备份并重建本地 Git 仓库')
+            if not self._rebuilding_repository and has_missing_object_failure:
+                log.warning('候选代码源导入时检测到本地对象缺失，开始自动备份并重建本地 Git 仓库')
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)